lms/djangoapps/instructor_task/tasks_helper.py

```diff
@@ -730,7 +734,8 @@ def upload_grades_csv(_xmodule_instance_args, _entry_id, course_id, _task_input,
                 header = [section['label'] for section in gradeset[u'section_breakdown']]
                 rows.append(
                     ["id", "email", "username", "grade"] + header + cohorts_header +
```

Comment:
This seems clearer to me:

    rows.extend([
        ["id", "email", "username", "grade"],
        header,
        cohorts_header,
        ....
    ])

I also don't like the way several items are on some lines, so it is hard to see how many items are being added.